rules/windows/process_creation/proc_creation_win_mshta_susp_execution.yml

```
@@ -34,7 +34,6 @@ detection:
             - '.doc'
             - '.zip'
             - '.dll'
```

```suggestion
            - '.dll'
            # - '.exe'
```
I think it's best to keep it in there so that the same mistake won't be done twice. (exe will match on the mshta.exe itself)